modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/IdPSuccessTest.java

```diff
@@ -529,7 +576,13 @@ public void addSAMLStandardBasedIdP() throws IOException {
                 .log().ifValidationFails()
                 .assertThat()
                 .statusCode(HttpStatus.SC_CREATED)
-                .header(HttpHeaders.LOCATION, notNullValue());
+                .header(HttpHeaders.LOCATION, notNullValue())
+                .body("name", equalTo(SAML_IDP_NAME))
+                .body(baseIdentifier + "authenticatorId", equalTo(SAML_AUTHENTICATOR_ID))
+                .body(baseIdentifier + "name", equalTo(new String(Base64.getDecoder().decode(SAML_AUTHENTICATOR_ID))))
+                .body(baseIdentifier + "isEnabled", equalTo(true))
+                .body(baseIdentifier + "definedBy", equalTo(DEFINED_BY_SYSTEM))
+                .body(baseIdentifier + "tags", Matchers.hasItems("SAML"));
```

Comment:
assert self as well

Reply:
This is updated with 7831a169607232cefc26307b20c8ca3746542a52